### PR TITLE
Added 2.1.2 release notes.

### PR DESCRIPTION
**Pull request by csterwa**

### CODE REVIEW

File: release-notes.html.md.erb [hunk 1]
```diff
@@ -24,6 +24,15 @@ For each client app, you must:
...
 * Update the build file to use the new versions of the Spring Cloud Services client dependencies (see [Client Dependencies](./common/client-dependencies.html))
 
+## <a id="2-1-2"></a>2.1.2
+
+**Release Date: 13th March 2020**
+
+Enhancements included in this release:
+
+* Fixed issue with custom domain not being applied to service broker's worker application
+* Custom buildpack is now used when deploying service broker applications
```
Comment (fifthposition) [inline release-notes.html.md.erb]: Are these both bug fixes rather than enhancements?

Comment (csterwa) [reply release-notes.html.md.erb]: Yes

**Comment by csterwa** [comment]
Merging for now and fixes can be applied later

**Comment by fifthposition** [comment]
Fixed. Sorry for delay!

**Pull request merged**